apps/omg_watcher/test/exit_processor/core_test.exs

```diff
@@ -353,7 +354,7 @@ defmodule OMG.Watcher.ExitProcessor.CoreTest do
              |> Core.invalid_exits(processor)
 
     # exit validly finalizes and continues to not emit any events
-    {:ok, {_, _, spends}, _} = State.Core.exit_utxos([@utxo_pos1], state)
+    {:ok, {_, _, spends}, _} = [@utxo_pos1] |> prepare_exit_finalizations() |> State.Core.exit_utxos(state)
```

Comment:
We don't need that, exit_utxos can handle an enum of Utxo.position.

Reply:
yes, but the purpose here is to mimic the behavior of `ExitProcessor` a little bit more. It's clunky, but I'd rather do that, than end up in a similar bug in the future